samples/python/requirements.txt

```
@@ -1,2 +1,2 @@
 opencv-python==4.5.*
-numpy>=1.16.6,<1.20
+numpy>=1.16.6,<=1.23.1
```

```suggestion
numpy>=1.21.6,<=1.23.1
```
@rkazants can it be tested? Add it in all relevant places.

```suggestion
numpy>=1.19,<=1.23.1
```
Or at least something like this would be great.

---

1. Eventually dropping support for Py3.6.
2. There are already known issues and bugs in older numpy releases. Moving it up makes environment simply safer. Please refer to discussion in ticket _70927_

---

PS Please don't re-request review when discussion is still on-going.

---

@jiwaszki, yes, it might be less safer to use `1.16` numpy but the safety of the environment is not our responsibility and user must care of his environment. We didn't see any problem with this lower bound for 5-6 releases or more and someone can still use old environment. We don't support it but it can potentially work on old environment. I am leaning to have OpenVINO to be capable installed on more environments and run there rather that `openvino-dev` package is no longer installed on old environment.